src/asm/charmap.c

```diff
@@ -58,15 +58,14 @@ static struct Charmap *charmap_Get(const char *name)
 	return hash_GetElement(charmaps, name);
 }
 
-static struct Charmap *resizeCharmap(struct Charmap *map, size_t capacity)
+static void resizeCharmap(struct Charmap **map, size_t capacity)
 {
-	struct Charmap *new = realloc(map, sizeof(*map) + sizeof(*map->nodes) * capacity);
+	*map = realloc(*map, sizeof(struct Charmap) + sizeof(struct Charnode) * capacity);
```

Comment:
Why changing the expressions to types?

Reply:
I felt that `*(*map)->nodes` is ugly.

Reply:
I agree, but I'd prefer it regardless, for semantic reasons.